ingest/Ingest.py

```diff
@@ -0,0 +1,162 @@
+__author__ = 'Hao'
+
+import multiprocessing
+from SPARQLWrapper import SPARQLWrapper, JSON
+import itertools
+import json
+import requests
+
+class Ingest:
+    """Helper class governing an ingest process."""
+
+    def __init__(self, object_index, object_type,
+                 get_objects_query_location, describe_object_query_location, variable_name_sparql):
+        """
+        Constructor method of the Ingest class.
+        :param object_index:                    object index
```

Comment:
while we change object_index -> elasticsearch_index, should we also change:
object_type -> elasticsearch_type?